package.json

```diff
@@ -21,21 +21,23 @@
     "@types/node": "20.4.10",
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
-    "appwrite": "11.0.1",
+    "appwrite": "^11.0.1",
     "autoprefixer": "10.4.14",
     "eslint": "8.47.0",
     "eslint-config-next": "13.4.13",
     "next": "13.4.13",
     "next-themes": "^0.2.1",
     "nookies": "^2.5.2",
+    "pnpm": "^8.14.0",
     "postcss": "8.4.31",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-feather": "^2.0.10",
     "react-redux": "^8.1.2",
     "react-toastify": "^9.1.3",
     "tailwindcss": "3.3.3",
-    "typescript": "5.1.6"
+    "typescript": "5.1.6",
+    "yarn": "^1.22.21"
```

Comment:
yarn should be a global dependency and not be added into the project. Please remove it as well